### PR TITLE
build(yarn): Upgrade from yarn 1.x to yarn 2.x

### PR DESCRIPTION
**Pull request by TarikGul**
Upgrade from yarn 1.x to yarn 2.x. To track similar discussions for our package migrations checkout [#518](https://github.com/paritytech/substrate-api-sidecar/pull/518).

### CODE REVIEW

**Review by emostov — APPROVED** [review]
🍓

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.22.10.cjs"
```
Comment (niklasad1) [inline .yarnrc]: dq: why is this needed? 

I thought this PR would get rid of yarn `1.x` ^^

Comment (TarikGul) [reply .yarnrc]: Nice catch!!! It actually does, but as I was updating the pathing didnt get upgraded properly! Probably because of the order I did `yarn set version <version>` in.

Comment (TarikGul) [reply .yarnrc]: Fixed! Thanks for the catch.

**Review by niklasad1 — APPROVED** [review]
